coding-guides/tracking-used-invites.md

```diff
@@ -8,48 +8,73 @@ description: >-
 
 A relatively frequent thing people would love to know, is "what invite someone used to join". Unfortunately, the Discord API does not provide the information about the invite used to join a server.
 
-To get around this, we need to do a couple separate steps. The first one is to go into your bots application page and enable the `GUILD_MEMBERS` privileged intent and add it to your bots client, then the second thing is to fetch all of the invites for each guild and store it in a temporary object. The third is to add, and remove invites from the object via the `inviteCreate` and `inviteDelete` events and finally is to fetch a guild's invite whenever someone joins, and find which one has a new use on it.
+To get around this, we need to do two separate steps. The first one is to fetch all of the invites for each guild and store it in a temporary Map. The second is to fetch a guild's invite whenever someone joins, and find which one has a new use on it. Thankfully, that's actually pretty simple!
 
-Thankfully, that's actually pretty simple!
+{% hint style="info" %}
+If you don't know what a Map is, read up [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)
+{% endhint %}
 
-While the below is a fair approximation of invite tracking, it's still not perfect. As it doesn't track single use invites \(when right-clicking someone, and doing Invite To =&gt; Server\). Those exist only for a few moments and cannot be tracked at all.
+While the below is a fair approximation of invite tracking, it's still not perfect. There are 2 things it doesn't track:
+
+* Temporary one-use invites \(when right-clicking someone, and doing Invite To =&gt; Server\). Those exist only for a few moments and cannot be tracked at all.
+* Invites created after the bot loaded. That would require fetching on a loop which is dangerous for API spam.
```

Comment:
Because of the new events this statement is incorrect, the invites created after bot loading are tracked.
```suggestion

```

Reply:
Done!